### PR TITLE
finished api-validations challenges

### PR DESCRIPTION
**Pull request by Punjabi2021**

### CODE REVIEW

File: app/models/cat.rb [hunk 1]
```diff
@@ -1,2 +1,5 @@
 class Cat < ApplicationRecord
+  validates :name, :age , :enjoys ,:image, presence: true
+  validates :enjoys ,length: {minimum:10}
+  validates :age ,numericality: {minimum:1}
```
Comment (sjproctor) [inline app/models/cat.rb]: Ooo fancy!

File: spec/models/cat_spec.rb [hunk 2]
```diff
@@ -1,5 +1,30 @@
...
+  it "should validate enjoys will have ten or more characters" do
+  cat = Cat.create
+  expect(cat.errors[:enjoys]).to_not be_empty
+  end
+
+  it "should validate image" do
+  cat = Cat.create
+  expect(cat.errors[:image]).to_not be_empty
+  end
+
```
Comment (sjproctor) [inline spec/models/cat_spec.rb:29]: Make sure you are indenting your code blocks.

File: app/models/cat.rb [hunk 3]
```diff
@@ -1,2 +1,5 @@
 class Cat < ApplicationRecord
+  validates :name, :age , :enjoys ,:image, presence: true
+  validates :enjoys ,length: {minimum:10}
```
Comment (sjproctor) [inline app/models/cat.rb]: Make sure you don't have extra spaces after your commas.

**Review by sjproctor — CHANGES_REQUESTED** [review]
Take a quick pass at your indentation and spaces.

File: app/models/cat.rb (hunk 3 above)
Comment (Punjabi2021) [reply app/models/cat.rb]: Yes ma'am

File: spec/models/cat_spec.rb (hunk 2 above)
Comment (Punjabi2021) [reply spec/models/cat_spec.rb:29]: Will be more diligent.

File: app/models/cat.rb (hunk 1 above)
Comment (Punjabi2021) [reply app/models/cat.rb]: Pinky's up Ma'am!

**Review by sjproctor — APPROVED** [review]
beautiful!

**Pull request merged**